libraries/AP_Airspeed/AP_Airspeed_NMEA.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "AP_Airspeed_Backend.h"
+#include <AP_HAL/AP_HAL.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+
```

Comment:
ifdef the class, so nobody can use it

Reply:
done

Reply:
ifdef should be on class definition

Reply:
I recall why I didn't do this, I am getting a compile error on linux,
```
Build failed
Traceback (most recent call last):
  File "/home/runner/ardupilot/modules/waf/waflib/Task.py", line 338, in process
    ret = self.run()
  File "Tools/ardupilotwaf/ap_library.py", line 162, in run
    raise Errors.WafError('%s: library header uses vehicle-dependent macros' % n.srcpath())
waflib.Errors.WafError: libraries/AP_Airspeed/AP_Airspeed_NMEA.h: library header uses vehicle-dependent macros
```
Maybe there is some other workaround?